iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/json/OutputDeclarationImplTest.java

```diff
@@ -98,6 +101,29 @@ void shouldParseOutputs() {
     assertThat((ArmTree) children.get(8)).is(STRING_LITERAL).has("value", "countValue").hasRange(7, 17, 7, 29);
   }
 
+  @ParameterizedTest
+  @CsvSource(delimiterString = ";", value = {
```

Comment:
```suggestion
  @ValueSource(string = {
```
Would be more appropriate